Migration-Guide.md

```diff
@@ -1,30 +1,64 @@
-To address a vulnerability in a dependency of Spectral, we updated this tool to use the latest major version of Spectral. This version does not support loading of static assets for custom rulesets so to maintain consistent behavior, we re-organized our repo and released our [custom ruleset as its own NPM package](https://www.npmjs.com/package/@ibm-cloud/openapi-ruleset). Due to these changes, a number of incompatible changes have been released in `v0.52.0`. These changes are outlined in this document, along with steps to migrate to this version.
+# Migration Guide
+This migration guide is intended to help users migrate from the version 0.x pre-release of the IBM OpenAPI Validator
+to the new version 1.0 official release.
 
-### IBM ruleset name change
-Formerly, our custom ruleset was named `ibm:oas`. Any user rulesets extending our base rules would “extend” this set. The same ruleset is available to extend but it is under a new name: `@ibm-cloud/openapi-ruleset`. The only change a user needs to make to migrate is change this name in their Spectral config file:
-```yaml
-# example .spectral.yaml file
+## Minimum Node version
+The minimum supported version of Node.js remains at version 14.x.
 
-# extends: ibm:oas -> this will no longer work
-extends: '@ibm-cloud/openapi-ruleset'
-rules:
-...
-```
+## API
+The v0.x pre-release validator supported an API to enable users to invoke the validator programmatically
+from a javascript application.
+This API is no longer supported in the v1.x validator; however, if there is a need to enable the
+validator to be invoked programmatically from the user's application, then it could be considered
+for a future release.
 
-The tool will not fail if you do not make this change, it will just ignore the user’s ruleset and proceed with the default (which is the IBM ruleset).
+## CLI Changes
+Changes were made to the command-line options supported by the validator:  
+1. In the v0.x pre-release validator, the `-c`/`--config` option was used to specify the name
+of the user's configuration file to use instead of `.validaterc`.  In the new v1.x validator,
+these same options are now used to specify the name of your validator configuration file, which
```

Comment:
Sounds more natural to remain singular like you started ("the `-c`/`--config` option")

```suggestion
this same option is now used to specify the name of your validator configuration file, which
```

Reply:
good catch